tools/addrundata/README.md

```diff
@@ -21,23 +21,19 @@ There are two modes of operation:
 
 [Rundata Check]: /.github/workflows/rundata_check.yml
 
-An example `rundata_test.go` looks like this:
+An example `metadata.textproto` looks like this:
 
-```go
-// Code generated by go run tools/addrundata; DO NOT EDIT.
-package foo_test
-
-import "github.com/openconfig/featureprofiles/internal/rundata"
+```
+# proto-file: proto/metadata.proto
```

Comment:
Should we fully-qualify the proto file with repo?

github.com/openconfig/featureprofiles/proto/metadata.proto